install_microshift.py

```diff
@@ -0,0 +1,120 @@
+import subprocess
+import time
+
+def microshift_iso_builder():
+    cmd = f"dnf install -y osbuild-composer composer-cli cockpit-composer bash-completion firewalld" 
+    output = subprocess.run(cmd, shell=True)
+    
+    cmd = f"systemctl enable osbuild-composer.socket cockpit.socket --now" 
+    output = subprocess.run(cmd, shell=True)
+    
+    uname_m= subprocess.check_output(['uname', '-m']).decode('utf-8').strip()
+
+    content = f'''id = "rhocp-4.14"\nname = "Red Hat OpenShift Container Platform 4.14 for RHEL 9"\ntype = "yum-baseurl"\nurl = "https://cdn.redhat.com/content/dist/layered/rhel9/{uname_m}/rhocp/4.14/os"\ncheck_gpg = true\ncheck_ssl = true\nsystem = false\nrhsm = true'''
+
+    with open('rhocp-4.14.toml', 'w') as file:
+        file.write(content)
```

Comment:
Same here. you write a file, and the next time you call this, the file is already there.

Reply:
I agree. We delete these files if they exist at the beginning of the program if they exist